bitcoin/src/bip152.rs

```diff
@@ -143,6 +143,8 @@ impl Decodable for ShortId {
     }
 }
 
+/// A set of things to relay.
```

Comment:
Maybe
`/// A structure to relay a block header, short IDs, and a select few transactions.`

Reply:
Lol, I must have been asleep when I did this PR, I literally could not think of howto say that in a sentence :)  Thanks man.